Source/ProgressionSystemRuntime/Public/LevelActors/PSStarActor.h

```diff
@@ -14,26 +14,50 @@ class PROGRESSIONSYSTEMRUNTIME_API APSStarActor : public AActor
 	// Sets default values for this actor's properties
 	APSStarActor();
 
+	/** Static helper function that plays any given star animation from various places. */
```

Comment:
Is not `static` anymore:)
just helper, or universal helper

Reply:
agree, https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/d9a88f9a6a5376ff19e6e0b6bcd5abd5d6b93e1c